docs/quick-lint-js.schema.config

```
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+
+    "title": "quick-lint-js configuration",
+    "description": "The quick-lint-js CLI and quick-lint-js editor plugins, can be configured using a quick-lint-js.config file.",
+    "$comment": "Refer https://quick-lint-js.com/config for more details.",
+
+    "type": "object",
+
+    "properties": {
+        "globals": {
+            "description": "Global variables which programs can use. It tells quick-lint-js what global variables to assume exist.",
+            "$comment" : "If the global variables you want are from a popular platform or library, you might want to use 'global-groups' instead.",
+            "type": "object",
+
+            "patternProperties": {
+                "^.*$": {
```

Global variables must not be empty:
```suggestion
                "^.+$": {
```